routes/users.js

```diff
@@ -0,0 +1,112 @@
+import express from 'express';
+import bcrypt from 'bcryptjs';
+
+import { addUserValidationRules, validateData } from '../validate.js';
+import { User } from '../models';
+
+const router = express.Router();
+
+const getUsers = async (request, response) => {
+  try {
+    const users = await User.findAll();
+    return response.status(200).send(users);
+  } catch (error) {
+    return response.status(400).send('An error occured.');
+  }
+};
+
+const addUser = async (request, response) => {
+  const { userName, email, firstName, lastName, password } = request.body;
+
+  try {
+    const foundUser = await User.findAll({
```

Comment:
You can use `sequelize's` findOne here and get an entity instead of a list that could be length either 1 or 0 since `userName` is unique.